### PR TITLE
48 cdk framework integration

### PR DESCRIPTION
**Pull request by bdemann**

### CODE REVIEW

File: examples/heartbeat/test/test.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { run_tests, Test } from 'azle/test';
+import { execSync } from 'child_process';
+import { get_tests } from 'azle/examples/heartbeat/test/tests';
+import { createActor } from './dfx_generated/heartbeat';
+
+const init_canister = createActor('rrkah-fqaaa-aaaaa-aaaaq-cai', {
```
Comment (lastmjs) [inline examples/heartbeat/test/test.ts:6]: This should be called `heartbeat_canister`

File: examples/heartbeat/src/main.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+initialized: bool = False
+
+@heartbeat
+def heartbeat_():
+    global initialized
+    initialized = True
+
+@query
+def getInitialized() -> bool:
```
Comment (lastmjs) [inline examples/heartbeat/src/main.py:11]: Can you make this camel case, and fix it in Azle as well?

File: examples/init/src/main.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    global user
+    global reaction
+    global owner
+
+    user = init_user
+    reaction = init_reaction
+    owner = init_owner
+
+@query
+def getUser() -> opt[User]:
```
Comment (lastmjs) [inline examples/init/src/main.py:25]: Can you change these all to camel case and fix them in Azle too?

File: examples/init/test/test.ts [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        {
+            name: `build canister ${canister_name}`,
+            prep: async () => {
+                execSync(`dfx build ${canister_name}`, {
+                    stdio: 'inherit'
+                });
+            }
+        },
+        {
+            name: `install canister ${canister_name}`,
```
Comment (lastmjs) [inline examples/init/test/test.ts:57]: We don't need all of these crazy things anymore in dfx 0.12.0-beta.x, we can just use dfx deploy

File: examples/inspect_message/src/main.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+@update
+def accessible() -> bool:
+    return True
+
+@update
+def inaccessible() -> bool:
+    return False
+
+@update
+def alsoInaccessible() -> bool:
```
Comment (lastmjs) [inline examples/inspect_message/src/main.py:25]: Camel case, here and in Azle

File: examples/inspect_message/test/test.ts [hunk 6]
```diff
@@ -0,0 +1,30 @@
+import { run_tests, Test } from 'azle/test';
+import { execSync } from 'child_process';
+import { get_tests } from 'azle/examples/inspect_message/test/tests';
+import { createActor } from './dfx_generated/inspect_message';
+
+const inspect_canister = createActor('rrkah-fqaaa-aaaaa-aaaaq-cai', {
```
Comment (lastmjs) [inline examples/inspect_message/test/test.ts:6]: Should be called `inspect_message_canister`

**Review by lastmjs — CHANGES_REQUESTED** [review]

File: examples/heartbeat/src/main.py (hunk 2 above)
Comment (bdemann) [reply examples/heartbeat/src/main.py:11]: https://github.com/demergent-labs/azle/pull/770

**Review by lastmjs — APPROVED** [review]

**Pull request merged**